app/scripts/controllers/evaluation/explore.js

```diff
@@ -1,24 +1,31 @@
 'use strict';
 
-angular.module('wcagReporter')
-  .controller(
-    'EvalExploreCtrl',
-    function ($scope, appState, $timeout, evalExploreModel) {
-      $scope.state = appState.moveToState('explore');
-      $scope.exploreModel = evalExploreModel;
+angular
+  .module('wcagReporter')
+  .controller('EvalExploreCtrl', function (
+    $scope,
+    appState,
+    $timeout,
+    evalExploreModel
+  ) {
+    $scope.state = appState.moveToState('explore');
+    $scope.exploreModel = evalExploreModel;
 
-      $scope.updateSpec = function (tech) {
-        if (techMap[tech.title]) {
-          tech.id = techMap[tech.title];
-        }
-      };
+    $scope.updateSpec = function (tech) {
+      if (techMap[tech.title]) {
+        tech.id = techMap[tech.title];
+      }
+    };
 
-      $scope.knownTech = angular.copy(evalExploreModel.knownTech);
-      $scope.otherTech = [];
+    $scope.knownTech = angular.copy(evalExploreModel.knownTech);
+    $scope.otherTech = [];
 
-      // set relied upon technologies in the right field
-      evalExploreModel.reliedUponTechnology.forEach(function (tech) {
+    // set relied upon technologies in the right field
+    evalExploreModel.reliedUponTechnology
+      .forEach(function (tech) {
         var index = $scope.knownTech
+          // Find exact matching index in knownTech of reliedUponTechnology
+          // it will be an user defined technology otherwise
           .reduce(function (index, currTech, currIndex) {
```

Comment:
IE11 unsupported Array method.

Which brings me to next points:

- At the time of first publication, I think the Reporting tool supported IE11?

- If we want to phase IE11 support out, which is fine by me (and Microsoft too), I want to do this so that our users are prepared for this and will get the time to move to another browser, e.g. the newer MS browser Edge if they are bound to MS products. An approach could be: deprecation / drop of support notice within time and prepare a no-IE release.

- If we should meet with the user that is confined to IE we should keep support until MS stops supporting it, which I have read is somewhere in 2025 see https://death-to-ie11.netlify.com/

Reply:
I dont want to add or create build/compile code to solve production problems. This might complicate / obfuscate parts of the code.

Also we started to call this a problem, but it is functioning code! 🎊